.env.local.example

```diff
@@ -0,0 +1 @@
+FONT_AWESOME_NPM_TOKEN=
```

Comment:
Maybe we could move this to `frontend/.env.local.example`, since it seems to only be needed for the frontend? Adding some instructions on where to find this env var in the README would be nice too